channel-router-setup-processor/package.json

```diff
@@ -3,42 +3,59 @@
   "version": "0.1.0",
```

Comment:
Can you update the lint-staged end of lines? There are multiple linting and formating errors on the repo. 

`"lint-staged": {
    "*.{js,ts}": [
      "eslint --fix",
      "prettier --write"
    ]
  }`

Reply:
This throws error - but I did run linting and fixed all the issues.